docs/desktop/gnome/screenshot.md

```diff
@@ -1,40 +1,65 @@
 ---
-title: Screenshot 
-author: Christine Belzie
-contributors: Steven Spencer, Ganna Zhyrnova  
+title: Taking Screenshots and Recording Screencasts in GNOME
+author: Wale Soyinka
+contributors:
+tags:
+  - gnome
+  - desktop
+  - screenshot
+  - screencast
+  - guide
 ---
-## Introduction
 
-Are you tired of using your keyboard to take images on your computer screen? Meet Screenshot, an application that helps you quickly take onscreen images.
+# **Taking Screenshots and Recording Screencasts in GNOME**
 
```

Comment:
Rinse and repeat as before: "Screenshot" comes down from the meta as a level 1 heading, and ## **Taking screenshots and recording screencasts in GNOME**